src/ray/common/id.cc

```diff
@@ -293,22 +303,6 @@ ObjectID ObjectID::GenerateObjectId(const std::string &task_id_binary, ObjectIDF
   return ret;
 }
 
-const TaskID GenerateTaskId(const JobID &job_id, const TaskID &parent_task_id,
```

Comment:
Just realized that we need still a deterministic way to generate task ids. Otherwise, lineage-based reconstruction won't work.

Reply:
Thanks, done.